services/web/server/src/simcore_service_webserver/utils.py

```diff
@@ -196,3 +196,13 @@ def get_tracemalloc_info(top=10) -> List[str]:
         )
 
     return top_trace
+
+# FUTURES
+def fire_and_forget_task(obj):
```

Comment:
MINOR: return ->None and try to define what obj is: any https://docs.python.org/3/library/typing.html#typing.Awaitable?